### PR TITLE
nhmusgs-gridmet container

### PR DESCRIPTION
**Pull request by ashalper-usgs**

### CODE REVIEW

File: nhmusgs-gridmet/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# entrypoint
+COPY --chown=nhm gridmet /usr/local/bin
+
+RUN chmod 744 /usr/local/bin/gridmet
+
+USER $USER
+
+WORKDIR /home/$USER
+
+# ...and remain running
```
Comment (isuftin) [inline nhmusgs-gridmet/Dockerfile]: This comment should be removed

Comment (ashalper-usgs) [reply nhmusgs-gridmet/Dockerfile]: OK, done.

File: nhmusgs-gridmet/gridmet [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```
Comment (isuftin) [inline nhmusgs-gridmet/gridmet]: Remove the space here

File: nhmusgs-gridmet/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Author - Andrew Halper
+#
+
+FROM nhmusgs/base
+
+LABEL maintainer="ashalper@usgs.gov"
+
+# check out source from repo.
+RUN git clone --depth=1 --branch v0.1.5 \
```
Comment (isuftin) [inline nhmusgs-gridmet/Dockerfile]: Set the branch value as an ARG above

See https://github.com/nhm-usgs/docker-images/blob/master/nhmusgs-nhm-prms/Dockerfile#L5-L8

Comment (ashalper-usgs) [reply nhmusgs-gridmet/Dockerfile]: OK, done.

File: nhmusgs-gridmet/gridmet (hunk 2 above)
Comment (ashalper-usgs) [reply nhmusgs-gridmet/gridmet]: This is valid, if old-timey. Believe it or not, there are some obscure, probably obsolete Unixes that cannot execute a shell script without the space there. CentOS is not one of them, so I'll remove it.

**Pull request merged**